build/executorch-config.cmake

```diff
@@ -24,20 +26,21 @@ target_include_directories(executorch INTERFACE ${_root})
 
 add_library(portable_kernels STATIC IMPORTED)
 find_library(
-    PORTABLE_KERNELS_PATH portable_kernels HINTS "${_root}"
+    PORTABLE_KERNELS_PATH portable_kernels
+    HINTS "${_root}"
+    CMAKE_FIND_ROOT_PATH_BOTH
 )
 set_target_properties(
     portable_kernels PROPERTIES IMPORTED_LOCATION "${PORTABLE_KERNELS_PATH}"
 )
 target_include_directories(portable_kernels INTERFACE ${_root})
 
 set(lib_list
-    etdump bundled_program extension_data_loader flatcc mpsdelegate
-    qnn_executorch_backend)
+    etdump bundled_program extension_data_loader flatcc mpsdelegate)
```

Comment:
oh is `qnn_executorch_backend` not needed anymore?

Reply:
Since `qnn_executorch_backend` dose not fully archive all required object files, I set the library dependency inside `examples/qualcomm/CMakeLists.txt` instead.